modules/ROOT/pages/errors/gql-errors/22N60.adoc

```diff
@@ -0,0 +1,11 @@
+= 22N60
+
+== Status Description
+error: data exception - illegal element. Encountered illegal `{ <<item>> }` element: `{ <<msg>> }`.
```

Comment:
When we use `{ <<msg>> }`, we omit the full stop. You'll need to update the codebase as well.
```suggestion
error: data exception - illegal element. Encountered illegal `{ <<item>> }` element. Reason: `{ <<msg>> }`
```